src/app/signin/signin.component.ts

```diff
@@ -0,0 +1,35 @@
+import { Component, OnInit, ViewEncapsulation } from '@angular/core';
+import { FormGroup, FormControl, Validators } from '@angular/forms';
+import { uniqueNamesGenerator, Config, names } from 'unique-names-generator';
+
+@Component({
+  selector: 'app-signin',
+  templateUrl: './signin.component.html',
+  styleUrls: ['./signin.component.scss'],
+  encapsulation: ViewEncapsulation.None,
+})
+export class SigninComponent implements OnInit {
+  constructor() {}
+
+  ngOnInit(): void {}
+  public registerForm: FormGroup = new FormGroup({
+    userName: new FormControl(),
```

Comment:
for single form control form group over kill. You can achieve same functionality with just ` FormControl`